wagtail/tests/settings.py

```diff
@@ -91,12 +91,12 @@
     'wagtail.tests.snippets',
     'wagtail.tests.routablepage',
     'wagtail.tests.search',
-    'wagtail.contrib.wagtailstyleguide',
-    'wagtail.contrib.wagtailsitemaps',
-    'wagtail.contrib.wagtailroutablepage',
-    'wagtail.contrib.wagtailfrontendcache',
-    'wagtail.contrib.wagtailapi',
-    'wagtail.contrib.wagtailsearchpromotions',
+    'wagtail.contrib.styleguide',
+    'wagtail.contrib.sitemaps',
+    'wagtail.contrib.routablepage',
+    'wagtail.contrib.frontendcache',
+    'wagtail.contrib.api',
+    'wagtail.contrib.searchpromotions',
```

Comment:
If there's no clear precedent across python/django then I have a slight preference for no-underscores, since it makes the task of updating import lines across a project more 'automatic' (less thinking required) and less annoying. But I guess this is [bikeshed](http://bikeshed.org/) territory, so I'll happily defer to the person who's actually doing the work :-)